recipes/fsleyes_props/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "fsleyes-props" %}
+{% set version = "1.6.3" %}
+{% set sha256 = "df5bac8e73da060754a4500ccb485bb4b97738381b2c56ab9bcd5fca0c56599a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - sphinx 1.6.*
+    - sphinx_rtd_theme 0.*
+    - mock 2.*
+    - coverage 4.*
+    - pytest 3.*
```

Comment:
Just to double check here, do you really need ``pytest`` to build your project?

Reply:
Unfortunately yes - this is an artifact of the way that I structured the `setup.py` dependencies for all of my projects - I listed documentation and testing dependencies in the `setup_requires` field.

At some point I'll adjust all of my projects so that this is not the case, but I was hoping to get everything into conda-forge before that :)